src/main/java/pl/joegreen/charles/model/topology/StarPopulationsTopology.java

```diff
@@ -0,0 +1,33 @@
+package pl.joegreen.charles.model.topology;
+
+import pl.joegreen.charles.model.Population;
+
+import java.util.Set;
+
+/**
+ * Created by FleenMobile on 2016-05-22.
+ */
+public class StarPopulationsTopology extends PopulationsTopology {
+
+    private Population populationInCenter = null;
+
+    @Override
+    public void add(Population population) {
+        populationsGraph.addVertex(population);
+
+        if (populationInCenter == null)
+            populationInCenter = population;
+        else {
+            populationsGraph.addEdge(populationInCenter, population);
+        }
+
+    }
+
+    @Override
+    public void removeOne() {
+        Set<Population> populations = populationsGraph.vertexSet();
+        if (populations.size() > 0) {
+            populationsGraph.removeVertex(populations.iterator().next());
```

Comment:
My bad. Thought about it, but forgot to add nullifying populationInCenter variable ;)